### DISCUSSION TITLE
Unable to install the tool

### DISCUSSION THREAD

**Discussion by CacheMeNot** [mgeeky/ProtectMyTooling#5] [General]
Hey there! I've been unable to install the tool. I've tried following the instructions listed, but I just can't get it to work. 

On Windows, I get a couple of FailedToImport and FileNotFound errors.
On Kali, it tells me that my syntax is incorrect (I'm using the syntax shown in the readme). Here, I've also tried running it using Powershell, but again it throws a syntax error.

Would really appreciate any guidance or pointers to where I'm messing up! Thank you!
![2022-08-20 17_14_15-Win10TestBox  Running  - Oracle VM VirtualBox](https://user-images.githubusercontent.com/55327342/185754552-864722c9-8aea-4610-be92-eee3e7799bc6.png)
![2022-08-20 20_06_27-Kali (Snap1)  Running  - Oracle VM VirtualBox](https://user-images.githubusercontent.com/55327342/185754575-2256cc26-dfd9-4d7a-875a-4bb6b5162d91.png)

**Comment by CacheMeNot** [comment]
Solved it, it was a silly mistake. 
using python instead of py in Windows & using python3 instead of py in kali fixed it.
The syntax mess was because I installed a weird py package, deleted it later.

**Reply by mgeeky** [reply]
Hi! I'm glad that's not my fault this time ^.^ 

Let me know if there's anything else I can do.

Regards,
Mariusz
